src/angular-maps/getting-started.md

```diff
@@ -76,17 +77,32 @@ export class TrainChooChooAppModule { }
 If you do not have specific configurations, you can use the standard configurations as well by using:
 
 ```ts
-import { EsriConfigModule } from '@sbb-esta/angular-maps';
+import { SbbEsriConfigModule } from '@sbb-esta/angular-maps';
 
 @NgModule({
   ...
-  imports: [EsriConfigModule],
+  imports: [SbbEsriConfigModule],
   ...
 })
 export class TrainChooChooAppModule {}
 ```
 
-### Step 3: Import the component modules
+### Step 3: Initialize the library configuration
+
+In your app.component.ts file (or any startup service), import the `SbbEsriLoaderService` and add as constructor parameter. Otherwise the configuration (the standard or your specific) won't initialize:
```

Comment:
Why is this necessary?
This seems to be better suited for a `forRoot` configuration.

Reply:
SbbEsriLoaderService is using this config (injected with SBB_ESRI_CONFIG_TOKEN). But SbbEsriLoaderService is not "injected" anywhere, so it's not "instantiated" automatically. Do you have any Idea how I can archive that?

Reply:
You should be able to instantiate the service directly in the forRoot method.

```ts

@NgModule({
  imports: [CommonModule],
  declarations: []
})
export class ExampleModule {
  static forRoot(toggle: boolean): ModuleWithProviders<ExampleModule> {
    return {
      ngModule: ExampleModule,
      providers: [
        {
          provide: SimpleService,
          useValue: new SimpleService({ message: 'example', toggle })
        }
      ]
    };
  }
}

```

https://stackblitz.com/edit/angular-ivy-cf3o73?file=src%2Fapp%2Fexample%2Fexample.module.ts

Alternatively you should also be able to apply this configuration in the forRoot method, although I haven't tested this yet.